chia/cmds/wallet.py

```diff
@@ -506,6 +506,9 @@ def nft_wallet_create_cmd(wallet_rpc_port: Optional[int], fingerprint: int) -> N
     show_default=True,
     callback=validate_fee,
 )
+@click.option(
+    "-rp", "--royalty-percentage", help="NFT royalty percentage in basis points", type=int, default=0, show_default=True
```

Comment:
I'd recommend adding an example in the help text, as it may still not be clear to the user what basis points are.